tests/phpunit/AutoReview/EnvChecker/EnvManipulatorsTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * This code is licensed under the BSD 3-Clause License.
+ *
+ * Copyright (c) 2017, Maks Rafalko
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * * Redistributions of source code must retain the above copyright notice, this
+ *   list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * * Neither the name of the copyright holder nor the names of its
+ *   contributors may be used to endorse or promote products derived from
+ *   this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\AutoReview\EnvChecker;
+
+use Infection\Tests\Env\BackupEnvVariables;
+use PHPUnit\Framework\TestCase;
+use function Safe\file_get_contents;
+use function Safe\sprintf;
+use function strpos;
+
+final class EnvManipulatorsTest extends TestCase
+{
+    /**
+     * @dataProvider \Infection\Tests\AutoReview\EnvChecker\EnvTestCasesProvider::envTestCaseTupleProvider
+     */
+    public function test_the_test_cases_manipulation_environment_variables_uses_the_backup_env_trait(
+        string $testCaseClassName,
+        string $fileWithEnvManipulations
+    ): void {
+        $this->assertNotFalse(
+            strpos(
+                file_get_contents($fileWithEnvManipulations),
+                'use BackupEnvVariables;'
+            ),
+            sprintf(
+                <<<'TXT'
+Expected the test case "%s" to be using the "%s" trait as environment variable manipulations have 
+been found in the file "%s".
+TXT
+                ,
+                $testCaseClassName,
+                BackupEnvVariables::class,
+                $fileWithEnvManipulations
+            )
+        );
```

Comment:
Is there any chance we can stop using `assertFalse()`, `assertTrue()`, and its counterparts, and instead start using dedicated assertions?

```suggestion
        $import = sprintf(
            'use %s;',
            BackupEnvVariables::class
        );

        $this->assertContains($import, file_get_contents($fileWithEnvManipulations), sprintf(
            <<<'TXT'
Expected the test case "%s" to be using the "%s" trait as environment variable manipulations have
been found in the file "%s".
TXT
            ,
            $testCaseClassName,
            BackupEnvVariables::class,
            $fileWithEnvManipulations
        ));
```

Reply:
When an acceptable alternative is available sure :)